forge-game/src/main/java/forge/game/trigger/TriggerExiled.java

```diff
@@ -77,6 +79,35 @@ public final boolean performTest(final Map<AbilityKey, Object> runParams) {
             return false;
         }
 
+        if (hasParam("WhileKeyword")) {
```

Comment:
would it make sense to extract the whole part into a function in the Trigger class?

Reply:
I think it might – WotC will probably make more triggers like this

Reply:
I think that's a great idea... I'll mark it for follow-up

Trying to get this thing "good enough for now" so we can finally merge it